bin/bootstrap_webserver

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+bin/rails db:create db:migrate
+
+bin/rails s -e production -p 3000
```

Comment:
How does the `-e production` parameter interact with the `RAILS_ENV=development` in docker-compose.yml?

Reply:
The RAILS_ENV is overwritten; this should be deleted. Good catch!